client/src/main/kotlin/io/spine/examples/chatspn/desktop/Client.kt

```diff
@@ -308,29 +315,63 @@ public class DesktopClient(
      *
      * @throws IllegalStateException if the user has not been authenticated
      */
-    public fun readChats(): List<ChatPreview> {
+    public fun readChats(): List<ChatCard> {
         checkNotNull(authenticatedUser) { "The user has not been authenticated" }
+        val byViewerFilter = QueryFilter.eq(
+            ChatCard.Column.viewer(),
+            authenticatedUser!!.id
+        )
         val chats = clientRequest()
-            .select(UserChats::class.java)
-            .byId(authenticatedUser!!.id)
-            .run()[0]
-        return chats.chatList
+            .select(ChatCard::class.java)
+            .where(byViewerFilter)
+            .run()
+        return chats
+    }
+
+    /**
+     * Returns personal chats where the provided user is a member.
+     *
+     * @param user ID of the user whose personal chats to read
+     */
+    public fun readPersonalChats(user: UserId): List<ChatCard> {
+        val byViewerFilter = QueryFilter.eq(
+            ChatCard.Column.viewer(),
+            user
+        )
+        val byTypeFilter = QueryFilter.eq(
+            ChatCard.Column.type(),
+            ChatType.CT_PERSONAL
+        )
+        val chats = clientRequest()
+            .select(ChatCard::class.java)
+            .where(byViewerFilter, byTypeFilter)
+            .run()
+        return chats
     }
 
     /**
      * Observes chats of the authenticated user.
      *
-     * @param onUpdate will be called when the user's chats updated
+     * @param onUpdate will be called when some user chat is updated or when a user joins a new chat
+     * @param onDelete will be called when the user ceases to be a member of a chat
```

Comment:
Please rephrase this "ceases". I don't think that how you would really speak about this case.